develop/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: Home
+version: develop
+latest: false
+---
+
+This workflow can be used to perform expression quantification for multiple
+BAM files. Expression levels will be determined for each BAM file/sample
+and will be merged together into a single table including all samples.
+
+Expression quantification will be performed using
+[StringTie](https://ccb.jhu.edu/software/stringtie/) and
+[HTSeq-Count](http://htseq.readthedocs.io/en/master/count.html).
+
+This workflow is part of [BioWDL](https://biowdl.github.io/)
+developed by [the SASC team](http://sasc.lumc.nl/).
+
+## Usage
+This workflow can be run using
+[Cromwell](http://cromwell.readthedocs.io/en/stable/):
+```bash
+java -jar cromwell-<version>.jar run -i inputs.json multi-bam-quantify.wdl
+```
+
+### Inputs
+Inputs are provided through a JSON file. The minimally required inputs are
+described below and a template containing all possible inputs can be generated
+using Womtool as described in the
+[WOMtool documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
+See [this page](/inputs.html) for some additional general notes and information
+about pipeline inputs.
+
+```json
+{
+  "MultiBamExpressionQuantification.bams": "A list of BAM files and sample identifiers (see 'BAM file input' below)",
+  "MultiBamExpressionQuantification.strandedness": "The strandedness of the samples: FR (forward-reverse), RF (reverse-forward) or None",
+  "MultiBamExpressionQuantification.outputDir": "The path to the output directory.",
+  "MultiBamExpressionQuantification.referenceGtfFile": "The path to the annotations GTF file. If not specified, Stringtie will be run unguided and the GTF file it produces will be used for HTSeq-Count",
+}
+```
+
+#### BAM file input
+BAM files need to be given as a list with one item per sample. Each of the
+items should be an object containing a `"Left"` element (the sample id) and a
+`"Right"` element (the BAM file and its index) following the structure as shown
```

Comment:
Left and right is very obscure. This is not very nice. Maybe we should invent a struct for this? 
For now this is ok.